packages/language-server/src/test/completion.test.ts

```diff
@@ -47,17 +53,13 @@ function assertCompletion(
     // eslint-disable-next-line  @typescript-eslint/restrict-template-expressions
     `Expected ${expected.items.length} suggestions and got ${
       completionResult.items.length
-    }: ${JSON.stringify(completionResult.items, undefined, 2)}`,
+    }: ${JSON.stringify(completionResult.items, undefined, 2)}`, // TODO: This is missing the output of `expected.items` so one can compare
   )
 
   assert.deepStrictEqual(
-    completionResult.items.map((items) => items.label),
-    expected.items.map((items) => items.label),
-  )
-
-  assert.deepStrictEqual(
-    completionResult.items.map((item) => item.kind),
-    expected.items.map((item) => item.kind),
```

Comment:
Looks like the item.kind assertion is gone? Is that intended?

Reply:
Not really. Let me add it back. (Although these asserts are terrible - only one can trigger... meaning the `length` one will never really trigger this way, but if we put it first, we do not get any good information about how/why it broke)

Reply:
Uh, this was actually really broken - the test assumed the wrong completion type. Great catch!